src/meta/test/backup_test.cpp

```diff
@@ -705,13 +722,13 @@ void meta_service_test_app::backup_service_test()
             std::string hint_message = fmt::format(
                 "backup interval must be greater than cold_backup_checkpoint_reserve_minutes={}",
                 meta_svc->get_options().cold_backup_checkpoint_reserve_minutes);
-            ASSERT_TRUE(resp.err == ERR_INVALID_PARAMETERS);
-            ASSERT_TRUE(resp.hint_message == hint_message);
+            ASSERT_EQ(ERR_INVALID_PARAMETERS, resp.err);
+            ASSERT_EQ(hint_message, resp.hint_message);
             req.backup_interval_seconds = old_backup_interval_seconds;
         }
 
-        // case3: backup policy contains valid app_id
-        // result: add backup policy succeed
+        // case3: backup policy contains valid and invalid app_id
```

Comment:
Unit tests are better to be implemented using GTEST_F, maybe you can refacor it in further pull request.